src/components/color-picker/color-picker.tsx

```diff
@@ -41,8 +41,8 @@ type Props = {
 
 function ColorPicker({ color = 0, small, onChange, colorList = COLORS }: Props) {
```

Comment:
I wonder if we should deprecate this one, given that:

- It is relying on a deprecated component (technically not a reason for deprecating it, as it should be re-implemented internally to use something else)
- More importantly: I think we should create a new one eventually that will probably have a different interface), and I'd also rename the new one to be a "select" and not a "picker".

Anyway, maybe not do anything about it now. I'm just dumping my thoughts.

Reply:
Logged under https://github.com/Doist/reactist/issues/682 👍